src/main/java/core/basesyntax/serviceimpl/CsvFileWriterServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.CsvFileWriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class CsvFileWriterServiceImpl implements CsvFileWriterService {
+    @Override
+    public void writeDataToFileCsv(Map<String, String> lines, String filePath) {
+
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(filePath))) {
+            for (Map.Entry<String, String> entry : lines.entrySet()) {
+                writer.write(entry.getKey() + "," + entry.getValue());
```

Comment:
Create constant for ","
```suggestion
                writer.write(entry.getKey() + DELIMITER + entry.getValue());
```